From 09a1f18cf9864dba35db34744200f7e40b401d6f Mon Sep 17 00:00:00 2001
From: Elena Grandi <valhalla-d@trueelena.org>
Date: Wed, 17 Aug 2016 19:39:21 +0200
Subject: Default on using gpg1 as gpgbinary

---
 gnupg.py      | 2 +-
 test_gnupg.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnupg.py b/gnupg.py
index 2a08d96..1d32a48 100644
--- a/gnupg.py
+++ b/gnupg.py
@@ -706,7 +706,7 @@ class GPG(object):
     }
 
     "Encapsulate access to the gpg executable"
-    def __init__(self, gpgbinary='gpg', gnupghome=None, verbose=False,
+    def __init__(self, gpgbinary='gpg1', gnupghome=None, verbose=False,
                  use_agent=False, keyring=None, options=None,
                  secret_keyring=None):
         """Initialize a GPG process wrapper.  Options are:
diff --git a/test_gnupg.py b/test_gnupg.py
index 3559e62..806a7bf 100644
--- a/test_gnupg.py
+++ b/test_gnupg.py
@@ -24,7 +24,7 @@ ALL_TESTS = True
 
 logger = logging.getLogger(__name__)
 
-GPGBINARY = os.environ.get('GPGBINARY', 'gpg')
+GPGBINARY = os.environ.get('GPGBINARY', 'gpg1')
 
 KEYS_TO_IMPORT = """-----BEGIN PGP PUBLIC KEY BLOCK-----
 Version: GnuPG v1.4.9 (MingW32)
